pkg/webreg/webreg.go

```diff
@@ -669,6 +680,16 @@ const imageExampleBaseConfig = `- as: e2e-steps
     post:
     - chain: ipi-deprovision
 `
+const imagePromotionConfig = `test_binary_build_commands: go test -race -c -o mytests # will create the test-bin tag
```

Comment:
This needs to be added to the data map: https://github.com/openshift/ci-tools/pull/457/files#diff-95aa6a9a4f1daf516567fb2eb073cb3fR937-R942